tests/MongoDB.Driver.Tests/Specifications/retryable-writes/TestRunner.cs

```diff
@@ -128,25 +169,21 @@ private IRetryableWriteTest CreateExecutableTest(BsonDocument operation)
             return executableTest;
         }
 
-        private IDisposable ConfigureFailPoint(BsonDocument failPoint)
+        private IDisposable ConfigureFailPoint(IMongoClient client, BsonDocument failPoint)
         {
             if (failPoint == null)
             {
                 return null;
             }
             else
             {
-                var adminDatabase = DriverTestConfiguration.Client.GetDatabase("admin");
-                var enableFailPointCommand = new BsonDocument
-                {
-                    { "configureFailPoint", "onPrimaryTransactionalWrite" }
-                }
-                .AddRange(failPoint);
-                adminDatabase.RunCommand<BsonDocument>(enableFailPointCommand);
+                var adminDatabase = client.GetDatabase("admin");
+                adminDatabase.RunCommand<BsonDocument>(failPoint);
```

Comment:
The failpoint document in the tests now consists of the ENTIRE command to send to the server.